admin/src/app/home/orders/stock-orders/receive/receive.component.ts

```diff
@@ -1,27 +1,36 @@
-import {Component, ElementRef, OnDestroy, OnInit, ViewChild} from '@angular/core';
```

Comment:
We use single quote (') wherever possible 
Avoid use of double quotes .

Turn off auto format on save 
Use tslint to format files